ui/packages/consul-ui/app/components/topology-metrics/index.hbs

```diff
@@ -17,9 +17,10 @@
     </div>
   {{#each @topology.Downstreams as |item|}}
     <TopologyMetrics::Card
-      @item={{item}}
+      @nspace={{@nspace}}
+      @dc={{@dc}}
```

Comment:
Can upstreams or downstreams be in a completely different dc/nspace to the origin service you are viewing? If so you'll have to grab the dc/nspace from the current downstream/upstream itself and not use the one from the origin server.

Reply:
Yes, that dc/nspace for each downstreams is within the @topology.Downstream and @topology.Upstreams items. I don't think I need to sent the Service dc/nspace in this case.

Reply:
oh, yes. For styling reasons I need to have the Service nspace/dc send down to each card. For example, does the downstream share the same nspace as the Service? If not, show the nspace with icon and tooltip.